cli/src/config/utils.ts

```diff
@@ -6,46 +6,69 @@ import {
 import { homedir } from 'os';
 import { resolve } from 'path';
 
-export const configDirectory = resolve(homedir(), '.hydra-cli');
+export const configDirectory = resolve(homedir(), '.subsquid-cli');
 export const configFilePath = resolve(configDirectory, 'config.json');
 
 export const defaultApiUrl = 'https://saas.infra.gc.subsquid.io/api';
 
+export type Config = {
+    apiUrl: string
+    credentials: string
```

Comment:
This will be refactored soon.
Credentials is valid only against that API url. 

We we eventually have multiple contexts
```ts
{
  currentContext: "dev",
  contexts : {
   {
     name: "production"
     apiUrl: "https://subsquid.io",
     token: "TOKEN_A"
    },
    {
     name: "dev",
     apiUrl: "https://dev.subsquid.io",
     token: "TOKEN_B"
    }  
  ]
}
```